src/control/server/config.go

```diff
@@ -47,6 +48,8 @@ const (
 	msgConfigNoServers  = "no servers specified in config"
 )
 
+type networkDeviceValidation func(string, string, uint) (error)
```

Comment:
unnecessary parenthesis around return type

Reply:
Fixed, and found/fixed a few others.